src/Store/SessionStore.php

```diff
@@ -38,25 +38,36 @@ class SessionStore implements StoreInterface
     protected $state = [];
 
     /**
-     * Create a store from the given request getting any initial state from the session of the request
+     * @param Member $member
+     */
+    public function __construct(Member $member = null)
+    {
+        if (!is_null($member)) {
```

Comment:
Does it matter if you set it as null? I realise it'll also reset the method and data, but that's OK during construction right?

Reply:
I guess not - this was a knee-jerk reaction to the `setMember()` method breaking when passed `null`. I'll tweak it for brevity 👍

Reply:
On second thoughts, this is still more brief than modifying `setMember(Member $member)`, as we can't use nullable typehints 😞

Reply:
Can't because its nullable hehe, merging